WordPress/src/main/java/org/wordpress/android/util/UrlUtilsWrapper.kt

```diff
@@ -18,4 +18,8 @@ class UrlUtilsWrapper @Inject constructor() {
     fun addUrlSchemeIfNeeded(url: String, addHttps: Boolean): String {
         return UrlUtils.addUrlSchemeIfNeeded(url, addHttps)
     }
+
+    fun getHost(urlString: String?): String {
```

Comment:
It seems `UrlUtils.getHost` can return null (result of `return uri.getHost();` is nullable). We probably either want to return `String?` or we might simply consider removing this method as it's not used anymore. Wdyt?

Reply:
I am going to remove this because it's no longer used. 

> It seems UrlUtils.getHost can return null (result of return uri.getHost(); is nullable).
Yes, it is indeed nullable but they are doing a null check before accessing it so doesn't that negate its the ability for the result to be null? or can the state of it still be changed? Let me know.

Reply:
>but they are doing a null check before accessing it 

You are absolutely right. I'm sorry, I missed it;).